src/client/src/ui/status-connection/styled.tsx

```diff
@@ -0,0 +1,120 @@
+import _ from 'lodash'
+import { styled } from 'rt-theme'
+import { FunctionComponent } from 'react'
+import React from 'react'
+import { Popup } from 'rt-components'
+import { ServiceConnectionStatus } from 'rt-types'
+
+const buttonHeight = '2rem'
+
+export const Button = styled.div`
+  background-color: ${({ theme }) => theme.core.lightBackground};
+  border-radius: 15rem;
+  user-select: none;
+  display: flex;
+  align-items: center;
+  justify-items: center;
+  cursor: pointer;
+  padding: 0 0.7rem;
+  height: 1.6rem;
+  font-size: 0.65rem;
+  font-weight: 350;
+`
+
+const StatusCircleCore: FunctionComponent<{ className?: string }> = ({ className }) => {
```

Comment:
Maybe the StatusCircleCore should not be in this file. I assume that the file is only about emotionJS style components.

Reply:
Ah I knew we had styled.tsx somewhere :-) Yeah I agree with the above.

Reply:
Agreed, moving it.